nextroute/model_sanity_check.go

```diff
@@ -161,3 +171,13 @@ func NewModelSanityCheck(model Model) ModelSanityCheck {
 	connect.Connect(con, &newModelSanityCheck)
 	return newModelSanityCheck(model)
 }
+
+// SanityCheckReport is the sanity check of a model.
+func SanityCheckReport(
+	ctx context.Context,
+	model Model,
+	duration time.Duration,
+	sanityCheckDepth SanityCheckDepth) (schema.Output, error) {
```

Comment:
```suggestion
	sanityCheckDepth SanityCheckDepth,
) (schema.Output, error) {
```

Since all the args are in a new line, I would prefer this. Very subjective though.